.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+  "recommendations": [
+    "editorconfig.editorconfig",
+    "formulahendry.dotnet-test-explorer",
```

Comment:
Personally, I do not recommend it as:
1. It slows down the VS Code.
2. Has quite of an [issues](https://github.com/formulahendry/vscode-dotnet-test-explorer/issues)
3. Is not maintained by any known/trusted organization
4. I find the test integration in the C# extension good enough

Reply:
Reduced this list to just EditorConfig, C#, and Markdownlint.